vault/dendron.topic.config.dendron.md

```diff
@@ -108,6 +108,12 @@ See [[workspace sync configuration options|dendron.ref.workspace#configuration-o
 
 Sets the [[synchronization strategy|dendron.ref.workspace#Workspace: Sync]] for [[workspace vaults|dendron.ref.vaults#workspace-vault]]. For regular vaults, please use the [[per-vault configuration|dendron.ref.vaults#sync]].
 
+### autoFoldFrontmatter
+- default: false
```

Comment:
@SeriousBug - did we want to consider defaulting this to true? Seems like the right move given most people don't want to mess around with the frontmatter and doing so is prone to cause errors :)